### PR TITLE
Allow removal of rel for Terms and Privacy skin objects

### LINKED ISSUE #5047: DNN Platform\Website\admin\Skins\Privacy.ascx.cs & Terms.ascx.cs
By default the links to privacy and terms pages are getting a rel="nofollow" added from the CS code. There are mixed opinions on having it set to nofollow (pro and contra). Some say having it set to nofollow could look kind of shady to Google.

To change this, one currently has to recompile DNN and use a custom build.

To improve this I would suggest to remove it from the CS code and put the rel="nofollow" into the ascx files instead, where one could customize the behavior without recompiling.

I could submit this change and would be glad not having to make custom builds anymore.

Best regards & happy DNNing
Andy9999

### PR DESCRIPTION
**Pull request by Andy9999**
Set the rel attribute of the hyperlink only if there is either no value set in the skin or a non blank value defined in the skin. If it is set to rel="" in the skin, the hyperlink will be rendered without a rel attribute.

Fixes #5047

### CODE REVIEW

File: DNN Platform/Website/admin/Skins/Privacy.ascx.cs [hunk 1]
```diff
@@ -60,7 +60,8 @@ protected override void OnLoad(EventArgs e)
                 }
 
                 this.hypPrivacy.NavigateUrl = this.PortalSettings.PrivacyTabId == Null.NullInteger ? this._navigationManager.NavigateURL(this.PortalSettings.ActiveTab.TabID, "Privacy") : this._navigationManager.NavigateURL(this.PortalSettings.PrivacyTabId);
-                this.hypPrivacy.Attributes["rel"] = this.Rel;
+                if (!string.IsNullOrWhiteSpace(this.Rel))
+                    this.hypPrivacy.Attributes["rel"] = this.Rel;
```
Comment (bdukes) [inline DNN Platform/Website/admin/Skins/Privacy.ascx.cs:64]:
```suggestion
                if (!string.IsNullOrWhiteSpace(this.Rel))
                {
                    this.hypPrivacy.Attributes["rel"] = this.Rel;
                }

```

File: DNN Platform/Website/admin/Skins/Terms.ascx.cs [hunk 2]
```diff
@@ -60,7 +60,8 @@ protected override void OnLoad(EventArgs e)
                 }
 
                 this.hypTerms.NavigateUrl = this.PortalSettings.TermsTabId == Null.NullInteger ? this._navigationManager.NavigateURL(this.PortalSettings.ActiveTab.TabID, "Terms") : this._navigationManager.NavigateURL(this.PortalSettings.TermsTabId);
-                this.hypTerms.Attributes["rel"] = this.Rel;
+                if (!string.IsNullOrWhiteSpace(this.Rel))
+                    this.hypTerms.Attributes["rel"] = this.Rel;
```
Comment (bdukes) [inline DNN Platform/Website/admin/Skins/Terms.ascx.cs:64]:
```suggestion
                if (!string.IsNullOrWhiteSpace(this.Rel))
                {
                    this.hypTerms.Attributes["rel"] = this.Rel;
                }

```

**Review by bdukes — APPROVED** [review]

**Review by valadas — APPROVED** [review]
Looks good to me.

I moved the issue number from the title to the description so it auto-closes that issue.

**Comment by Andy9999** [comment]
I have added a summary to the Getter/Setter for the Rel attribute for the Privacy and Terms skin objects. ("Gets or sets the rel attribute of the Privacy/Terms skin object. Set it to an empty string to not render a rel attribute at all.").

I made that change on my local branch ("Fix-for-issue-#5047"). Can I still submit that now? Or do I need to make a new branch and new PR for that? What would be the best approach?

**Comment by bdukes** [comment]
You can use the same branch, but it needs to be a new PR

**Pull request merged**